### PR TITLE
Add sass-docs to functions, variables and mixins

### PR DESCRIPTION
**Pull request by dylanjustice**
Add Sass-doc markdown based comments 
Resolves Issue #5

- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [/] Automated tests are passing
- [/] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [ / Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

File: src/functions/_calc-col-width.scss [hunk 1]
```diff
@@ -1,3 +1,12 @@
+
+/// Dynamically calculate the column within the grid
+/// @param {number} $col Number of columns
+/// @param {*} $grid [$grid-grid] - Grid variable to use
+/// @see $grid-grid
+/// @example scss
+///     width: calc-col-width(6, $my-grid);
+
```
Comment (jstevenson) [inline src/functions/_calc-col-width.scss:8]: The empty lines after the documentation varies from 0-2 lines. This should be consistent.

File: src/functions/_calc-col-width.scss [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+/// Dynamically calculate the column within the grid
```
Comment (jstevenson) [inline src/functions/_calc-col-width.scss:2]: It looks like this is missing the word "width"

File: src/mixins/_clearfix.scss [hunk 3]
```diff
@@ -1,3 +1,15 @@
+/// Helper to clear floats in a grid layout.
+/// @example scss
+///     .element {
+///         @include clearfix;
+///     }
+///
+/// @output css
+///     .element::after {
+///     clear: both;
```
Comment (jstevenson) [inline src/mixins/_clearfix.scss:9]: Indenting

File: src/mixins/_media.scss [hunk 4]
```diff
@@ -1,3 +1,14 @@
+/// Generates media query with included content.
+/// @param {string} $breakpoint Named breakpoint
+/// @content Styles to apply when screen meets max-width in $breakpoints map
+/// @see $breakpoints
+/// @example scss
+/// &.-span-one {
```
Comment (jstevenson) [inline src/mixins/_media.scss:6]: Indenting

**Review by jstevenson — APPROVED** [review]

**Review by wintondeshong — APPROVED** [review]

**Comment by dylanjustice** [comment]
@all-contributors please add @jstevenson for review

**Comment by dylanjustice** [comment]
@all-contributors please add @wintondeshong for review

**Comment by dylanjustice** [comment]
@all-contributors please add @dylanjustice for doc

**Pull request merged**